docs/pages/components/message-box.md

```diff
@@ -325,15 +325,15 @@ To enable responsive behavior for the paddings use the following modifier classe
     <li><code>.fd-message-box__content--xl</code>: 3rem - min-width: 1440px</li>
 </ul>
 <br><br>
-On mobile devices the component should be in `cosy` mode. Add the `fd-bar--cosy` modifier class to the header and the footer. The buttons in the footer should also be in Cosy mode. The deafult mode for Buttons is `cosy` so no modifier classes are required.
+On mobile devices the component should be in `cosy` mode. Add the `fd-bar--cozy` modifier class to the header and the footer. The buttons in the footer should also be in Cosy mode. The deafult mode for Buttons is `cosy` so no modifier classes are required.
```

Comment:
replace all cosy with cozy, not just class name

Reply:
done